index.d.ts

```diff
@@ -41,8 +41,9 @@ declare class BeeQueue {
   checkHealth(): Promise<BeeQueue.HealthCheckResult>;
   checkHealth(cb: (counts: BeeQueue.HealthCheckResult) => void): void;
 
-  close(): Promise<void>;
   close(cb: () => void): void;
+  close(timeout?: number): Promise<void>;
+  close(timeout: number, cb: () => void): void;
```

Comment:
The `timeout` is technically optional here, so you can pass something like `close(timeoutOrNull, callback)`. I dunno if this works in typescript, though:
```suggestion
  close(timeout?: number, cb: () => void): void;
```

Reply:
TS won't allow an optional parameter (with the `?`) before a required one, but I changed this to:

```ts
close(timeout: number | undefined | null, cb: () => void): void;
```

Reply:
sgtm